### DISCUSSION TITLE
Build issue in Beta

### DISCUSSION THREAD

**Discussion by roybarber** [payloadcms/payload#8812] [Q&A] [answered]
Hey everyone. Spent a couple hours trying to sort this but I’m getting build errors with beta branch using the payload generated admin code. 

![IMG_9607](https://github.com/user-attachments/assets/78aeecaf-0997-4dae-9c68-f816b569aca5)

type error that no matter what I do I can’t get a build. Local dev works perfectly! 

I’m also currently in Turkey and can’t ask on discord, since it’s banned :(

**Comment by r1tsuu** [comment, marked as answer]
Read the breaking changes here, you need to tweak your types a bit https://github.com/payloadcms/payload/releases/tag/v3.0.0-beta.111

> I’m also currently in Turkey and can’t ask on discord, since it’s banned :(

Does VPN work there?

**Reply by roybarber** [reply]
Perfect thank you. Annoyingly I’d already done that in the page a while ago. 

I’m on holiday on the mobile so don’t have a VPN
